src/pathfinding_service/api.py

```diff
@@ -152,7 +155,22 @@ def post(self, token_network_address: str) -> Tuple[dict, int]:
                 )
             )
 
-        return {"result": paths}, 200
+        # Create a feedback token and store it to the DB
+        feedback_token = create_and_store_feedback_token(
+            pathfinding_service=self.pathfinding_service,
+            token_network_address=token_network.address,
+        )
+
+        return {"result": {"paths": paths, "feedback_token": feedback_token.id.hex}}, 200
```

Comment:
We could undo this change and merge the PR, and apply this change as soon as the client changes are ready.

Reply:
If we have one token for multiple paths, we'll have to be able to store different results for different token+path combinations. That's ok, it was just something I didn't think about before.

Reply:
We could also keep all keys on the top level and have `{"result": ..., "token": ...}`. Then we would save one level of nesting and be able to merge it without breaking compatibility.